### ISSUE TITLE
Publish pipeline doesn't build before publishing

### ISSUE THREAD

**Issue by BetaHuhn** [BetaHuhn/metadata-scraper#65]
Your publishing pipeline seems to be automated, but it doesn't look like it's actually building your package before publishing it.

_Originally posted by @utrolig in https://github.com/BetaHuhn/metadata-scraper/discussions/64_

**Comment by BetaHuhn** [comment]
> I don't know why, but I can't seem to report issues.

You should be able to select bug report when creating a new issue.

I will investigate the issue this evening. [Here's](https://github.com/BetaHuhn/metadata-scraper/blob/master/.github/workflows/release.yml) the GitHub Action which handles the release. I guess the build job shouldn't be separate from the release job and instead we should build the files right before running the release action.

**Comment by utrolig** [comment]
That seems about right. 
I think you could probably scrap the whole `build` section, and just add `npm run build` to the release step

```yaml
release:
    needs: [lint]
    name: Build and release
    runs-on: ubuntu-latest
    if: "!contains(github.event.head_commit.message, '[skip ci]')"
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Setup Node.js
        uses: actions/setup-node@v2
        with:
          node-version: 14
      - name: Cache node modules
        uses: c-hive/gha-npm-cache@v1
      - name: Install dependencies
        run: npm ci
      - name: Run build command:
        run: npm run build
      - name: Run Semantic Release
        run: npx semantic-release
        env:
          GITHUB_TOKEN: ${{ secrets.GH_PAT }}
          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
          GIT_AUTHOR_NAME: "BetaHuhn Bot"
          GIT_AUTHOR_EMAIL: "bot@mxis.ch"
          GIT_COMMITTER_NAME: "BetaHuhn Bot"
          GIT_COMMITTER_EMAIL: "bot@mxis.ch"
```

**Comment by BetaHuhn** [comment]
Yes exactly.

I will fix this in the evening when I am off work or if you want you can create a PR.

**Comment by BetaHuhn** [comment]
Fixed in #66, thanks @utrolig!

I will release a new version later which should hopefully include the built files.
